modules/getting-started/c_sample-app.adoc

```diff
@@ -0,0 +1,11 @@
+// Metadata created by nebel
+//
+// UserStory: As an RH SSO customer, I want to complete the initial configuration of my standalone server
+
+[id="sample-app"]
+= Security for a sample application
+//If the assembly covers a task, start the title with a verb in the gerund form, such as Creating or Configuring.
```

Comment:
As the note states, maybe use a gerund in the titles? E.g. Securing a sample application or Configuring security for a sample application.

Reply:
Actually, I ended up not including this file in the document.